src/templates/en/2019/contributors.html

```diff
@@ -297,12 +297,12 @@ <h1 class="title">
     Contributors
   </h1>
   <div class="team">
-      <img src="/static/images/avatars/1.jpg">
-      <img src="/static/images/avatars/8.jpg">
-      <img src="/static/images/avatars/4.jpg">
-      <img src="/static/images/avatars/0.jpg">
-      <img src="/static/images/avatars/11.jpg">
-      <img src="/static/images/avatars/9.jpg">
+      <img loading="lazy" src="/static/images/avatars/1.jpg">
+      <img loading="lazy" src="/static/images/avatars/8.jpg">
+      <img loading="lazy" src="/static/images/avatars/4.jpg">
```

Comment:
Wouldn't bother lazy loading these:
```suggestion
      <img src="/static/images/avatars/4.jpg">
```